build-farm/make-adopt-build-farm.sh

```diff
@@ -59,7 +59,14 @@ fi
 
 [ -z "$JAVA_TO_BUILD" ] && echo JAVA_TO_BUILD not defined - set to e.g. jdk8u
 [ -z "$VARIANT"       ] && echo VARIANT not defined - assuming hotspot && export VARIANT=hotspot
-[ -z "$FILENAME"      ] && echo FILENAME not defined - assuming "${JAVA_TO_BUILD}-${VARIANT}.tar.gz" && export FILENAME="${JAVA_TO_BUILD}-${VARIANT}.tar.gz"
+if [ -z "$FILENAME"   ]; then
+  if [ "${VARIANT}" = "temurin" ]; then
+     # I don't like this - perhaps we should override elsewhere to keep consistency with existing release names
+     echo FILENAME not defined - assuming "${JAVA_TO_BUILD}-hotspot.tar.gz" && export FILENAME="${JAVA_TO_BUILD}-hotspot.tar.gz"
```

Comment:
I wonder if we need to keep that naming scheme?  I guess so to be backward compatible?

Reply:
Yeah I'm going for "minimal viable product" at the moment - get the changes in with minmal risk of screwing anything up ... Which is still quite risky!

Reply:
(Also bear in mind this is purely a default - normally it will be defined in our jenkins processes)